_config/extensions.yml

```diff
@@ -4,3 +4,9 @@ name: extensible-search-extensions
 SearchForm:
   extensions:
     - 'SearchFormResultsExtension'
+ExtensibleSearchSuggestion:
```

Comment:
The readme says these are disabled by default - but they appear here. It _should_ be user selectable as to whether it's enabled...